src/containers/Tenant/Diagnostics/Consumers/Consumers.tsx

```diff
@@ -24,29 +28,41 @@ const b = block('ydb-consumers');
 
 interface ConsumersProps {
     path: string;
+    type?: EPathType;
 }
 
-export const Consumers = ({path}: ConsumersProps) => {
+export const Consumers = ({path, type}: ConsumersProps) => {
     const dispatch = useDispatch();
 
+    const isCdcStream = isCdcStreamEntityType(type);
```

Comment:
Yes, it is. and it can be replaced with new check. But I decided to left it as it is (with isCDCStream), because it is topics and stream specific tab, and data loading logic is different